File: PHP/uberVan.php
<?php
require_once('car.php');
class UberVan extends Car{
    public $typeCarAcepted;
    public $seatMaterial;

    public function __contruct($license, $driver,
    $typeCarAcepted, $seatMaterial){
        parent::__contruct($license, $driver);
        $this->typeCarAcepted = $typeCarAcepted;
        $this->seatMaterial = $seatMaterial;
    }
}

File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car{
    Map<String, Map<String, Integer>> typeCarAcepted;
    ArrayList<String> seatsMaterial;
    private Integer passenger;

    public UberVan(String license, Account driver,
    Map<String, Map<String, Integer>> typeCarAcepted,
    ArrayList<String> seatsMaterial){
        super(license, driver);
        /* this.typeCarAcepted = typeCarAcepted;
        this.seatsMaterial = seatsMaterial; */
    }  
    
    @Override
    public void setPassenger(Integer passenger) {
        if(passenger == 6){
            this.passenger = passenger;
        }else{
            System.out.println("Debe ser mayor a 6");
        }
    }
}
